.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,15 @@
 ### What does this PR do?
-
-A brief description of the change being made with this pull request.
+<!-- A brief description of the change being made with this pull request. -->
 
 ### Motivation
-
-What inspired you to submit this pull request?
+<!-- What inspired you to submit this pull request? -->
 
 ### Additional Notes
-
-Anything else we should know when reviewing?
+<!-- Anything else we should know when reviewing? -->
 
 ### Review checklist (to be filled by reviewers)
```

Comment:
I've never understood why we ask the checklist to be filled by reviewers. I think it makes more sense to be filled by whoever writes the PR. I think the checklist is mostly here are a reminder of the things not to forget

Reply:
Yes agree